### PR TITLE
[MIG][10.0] account_analytic_distribution module

### PR DESCRIPTION
**Pull request by cubells**
- Minor changes from version 9.0

cc @Tecnativa

### CODE REVIEW

File: account_analytic_distribution/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Copyright 2017 Vicent Cubells - <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Analytic Distribution",
+    "summary": "Distribute incoming/outcoming account moves to several "
+               "analytic accounts",
+    "version": "10.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "http://www.tecnativa.com",
```
Comment (pedrobaeza) [inline account_analytic_distribution/__manifest__.py]: https

File: account_analytic_distribution/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "account",
```
Comment (pedrobaeza) [inline account_analytic_distribution/__manifest__.py:17]: Also on analytic?

File: account_analytic_distribution/migrations/9.0.1.0.0/pre-migration.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline account_analytic_distribution/migrations/9.0.1.0.0/pre-migration.py]: Remove this file

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Little changes, but overall OK

**Comment by pedrobaeza** [comment]
Please fix this, @cubells 

@chienandalu @luismontalba please review

File: account_analytic_distribution/__manifest__.py (hunk 2 above)
Comment (cubells) [reply account_analytic_distribution/__manifest__.py:17]: account module already has analytic dependency. It's not necessary.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

**Review by luismontalba — COMMENTED** [review]
Testing PR on runbot I can mark the check for "Force percent" even logged with admin, having adviser rights and developer mode active. Is it an issue?

**Comment by pedrobaeza** [comment]
Are you on edit mode?

**Comment by cubells** [comment]
@luismontalba can you retest now?

**Comment by luismontalba** [comment]
I can mark the "Force percent" check now, but then I can create a distribution with two lines 75% and 24% ,and I think this check should avoid it. Isn't it?
I can't see the analytic account entries menu in adviser menu (with related permission activated)

**Comment by cubells** [comment]
@luismontalba 

yes, you are right. I'm working on it.

**Comment by cubells** [comment]
@luismontalba @chienandalu 

Can you retest?

**Comment by cubells** [comment]
runbot error is due to analytic_tag_dimension module.

**Comment by luismontalba** [comment]
The think is now I can't see the "Force percent" check ... can you?

**Comment by pedrobaeza** [comment]
Rebasing now should be green

**Comment by cubells** [comment]
@luismontalba force percent field is in company form. Now you can force percent by company.

**Comment by cubells** [comment]
@pedrobaeza 
It's green.

**Comment by pedrobaeza** [comment]
@luismontalba please give your final blessing.

**Review by luismontalba — APPROVED** [review]
The "Force percent" check works fine

**Pull request merged**